src/OutputPresenter.ts

```diff
@@ -0,0 +1,268 @@
+import * as mobx from "mobx";
+import { templates } from "./Components/Template";
+import type {
+  Mood,
+  TrackEl,
+  LengthEl,
+  DurationTypes,
+} from "./Components/Template";
```

Comment:
As a sidenote, your `template` file isn't a `Component` and so should live in a different place